### PR TITLE
Feature/home screens

### LINKED ISSUE #11: [Design V2] Home screen
## Resources
- [ ] https://www.figma.com/file/I4OOjT4VhsSberDIAZTkcH/Covid-Watch-App-v2?node-id=1321%3A14677
- [ ] https://www.figma.com/file/I4OOjT4VhsSberDIAZTkcH/Covid-Watch-App-v2?node-id=1335%3A53044

### PR DESCRIPTION
**Pull request by madim**
Closes https://github.com/covid19risk/covidwatch-android-en/issues/11

### CODE REVIEW

File: app/src/main/res/drawable/bg_my_possible_exposures.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <corners android:radius="2dp" />
+
+    <stroke
+        android:width="1dp"
+        android:color="@color/lightGray" />
+
+</shape>
```
Comment (madim) [inline app/src/main/res/drawable/bg_my_possible_exposures.xml:10]: rename file to `bg_exposure_summary.xml`

File: app/src/main/res/layout/fragment_home.xml [hunk 2]
```diff
@@ -1,121 +1,115 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.swiperefreshlayout.widget.SwipeRefreshLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
```
Comment (Apisov) [inline app/src/main/res/layout/fragment_home.xml:4]: It's better to use ` android:scrollbars="none"`

File: app/src/main/java/org/covidwatch/android/ui/home/HomeViewModel.kt [hunk 3]
```diff
@@ -26,11 +26,11 @@ class HomeViewModel(
...
 
-    private val _userFlow = MutableLiveData<UserFlow>()
-    val userFlow: LiveData<UserFlow> get() = _userFlow
+    private val _navigateToOnboardingEvent = MutableLiveData<Event<Unit>>()
+    val navigateToOnboardingEvent: LiveData<Event<Unit>> get() = _navigateToOnboardingEvent
 
-    private val _isRefreshing = MediatorLiveData<Boolean>()
-    val isRefreshing: LiveData<Boolean> get() = _isRefreshing
+    private val _exposureSummary = MutableLiveData<ExposureSummary>()
+    val exposureSummary: LiveData<ExposureSummary> get() = _exposureSummary
```
Comment (Apisov) [inline app/src/main/java/org/covidwatch/android/ui/home/HomeViewModel.kt]: Use `PreferenceStorage` for this. Check `ExposureNotificationViewModel` for an example. LiveData from the preferences get updates internally so we have latest values all the time.

Plus it uses `CovidExposureSummary`, a UI class for `ExposureSummary` class from the EN lib.

File: app/src/main/res/layout/fragment_home.xml [hunk 4]
```diff
@@ -1,121 +1,115 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.swiperefreshlayout.widget.SwipeRefreshLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (Apisov) [inline app/src/main/res/layout/fragment_home.xml:2]: I really don't like that we have whole screen scrollable including Toolbar. That's not that case we would need to hide toolbar and if we do we would need to do it more properly. 
Anyways, let's keep it for now

File: app/src/main/res/layout/fragment_home.xml [hunk 5]
```diff
@@ -1,121 +1,115 @@
...
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:paddingHorizontal="32dp"
-                android:layout_marginTop="16dp"
-                android:text=""
-                android:visibility = "gone"
-                android:textAlignment="center" />
-        </LinearLayout>
-    </ScrollView>
+        <View
```
Comment (Apisov) [inline app/src/main/res/layout/fragment_home.xml:106]: what's that for?

**Review by Apisov — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_home.xml (hunk 5 above)
Comment (madim) [reply app/src/main/res/layout/fragment_home.xml:106]: this view pushes powered by content to the bottom of screen if device has enough space

File: app/src/main/java/org/covidwatch/android/ui/home/HomeFragment.kt [hunk 6]
```diff
@@ -1,76 +1,36 @@
...
-                    findNavController().navigate(R.id.splashFragment)
-                }
-                is ReturnUser -> {
-                    updateUiForReturnUser()
-                }
-            }
+        homeViewModel.navigateToOnboardingEvent.observe(viewLifecycleOwner, EventObserver {
+            findNavController().navigate(R.id.splashFragment)
         })
+        homeViewModel.exposureSummary.observe(viewLifecycleOwner, Observer(::bindExposureSummary))
```
Comment (Apisov) [inline app/src/main/java/org/covidwatch/android/ui/home/HomeFragment.kt:33]: @madim I forgot to comment that.
Please, use in the future `LifecycleOwner` extensions. `observe` and `observeEvent`
```
with(viewModel) {
    observe(data) {
        ...
    }
    observeEven(event) { 
      ...
    }
}
```

**Review by Apisov — APPROVED** [review]

**Pull request merged**